policy/policy.go

```diff
@@ -38,57 +38,59 @@ const (
 )
 
 var (
-	emptyPolicy            Policy
-	emptyVersionedPolicies VersionedPolicies
+	// EmptyPolicy represents an empty policy
+	EmptyPolicy Policy
 
-	// defaultPolicies are the default policies
+	// EmptyVersionedPolicies represents an empty VersionPolicies
+	EmptyVersionedPolicies VersionedPolicies
+
+	// DefaultPolicies are the default policies
 	// TODO(xichen): possibly make this dynamically configurable in the future
-	defaultPolicies = []Policy{
-		{
-			Resolution: Resolution{Window: 10 * time.Second, Precision: xtime.Second},
-			Retention:  Retention(2 * 24 * time.Hour),
-		},
-		{
-			Resolution: Resolution{Window: time.Minute, Precision: xtime.Minute},
-			Retention:  Retention(30 * 24 * time.Hour),
-		},
+	DefaultPolicies = []Policy{
```

Comment:
Unexport this perhaps? This is only used inside the policy package. I'm worried about giving code outside this package the ability to mutate them.